python/sglang/srt/layers/attention/flashinfer_backend.py

```diff
@@ -612,6 +619,8 @@ def call_begin_forward(
                 self.num_qo_heads,
                 self.num_kv_heads,
                 self.head_dim,
+                q_data_type=self.q_data_type,
```

Comment:
QQ Why do you specify the data type here?

Reply:
I am trying to sync with @yzh119 on this; I was seeing a dtype mismatch in the BatchPrefillWithRaggedKVCacheWrapper plan function otherwise.

Reply:
I’ve fixe it with the hopper branch. cc @yzh119 I’ll land soon.